lib/rubocop/ast/node/mixin/transmitter_node.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module AST
+    # Common predicates for nodes that have receivers:
+    # `send`, `csend`, `def`, `defs`, `super`, `zsuper`
+    #
+    # @note this mixins expects `#receiver` to be implemented
+    module TransmitterNode
```

Comment:
In fact this could go in `MethodIdentifierPredicates`, as it's technically part of the method signature.